src/zimit/zimit.py

```diff
@@ -26,16 +23,30 @@
 import requests
 from tld import get_fld
 from warc2zim.main import main as warc2zim
+from zimscraperlib.logging import getLogger
 from zimscraperlib.uri import rebuild_uri
 
-DEFAULT_USER_AGENT = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.0 Safari/605.1.15"
+from zimit.__about__ import __version__
+
+DEFAULT_USER_AGENT = (
+    "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 "
+    "(KHTML, like Gecko) Version/17.0 Safari/605.1.15"
+)
+
+EXIT_CODE_WARC2ZIM_CHECK_FAILED = 2
+EXIT_CODE_CRAWLER_LIMIT_HIT = 11
+NORMAL_WARC2ZIM_EXIT_CODE = 100
+
+LOGGER_NAME = Path(__file__).parent.name
```

Comment:
I dont think this brings any value but does bring some confusion si we can probably use a constant (scraper name?)